atlasdb-client/build.gradle

```diff
@@ -37,7 +37,6 @@ dependencies {
   api 'com.google.code.findbugs:jsr305'
   api 'com.google.guava:guava'
   api 'com.google.protobuf:protobuf-java'
-  api 'jakarta.annotation:jakarta.annotation-api'
```

Comment:
This is the only `api` dependency removed in this PR. I think it is unlikely that removing this will block upgrades - there's not much in this package that consumers would be using.

Reply:
yeah this should be fine. Very unlikely someone is getting it through us here.